examples/config.yaml

```diff
@@ -45,6 +45,8 @@ tools:
     repo: golang-demo
     api:
       name: trello
+      category: trello-github-integ
+      ident: trello-github-integ-golang-demo
```

Comment:
What does ident mean here? Ident normally means identification; does it mean identification here? Is the word used by Trello official doc so we use the abbreviation？More comments in the readme of this plugin would help.